Doc/Runway track changes.txt

```diff
@@ -38,4 +38,5 @@ Enniskillen         EGAB    2012    Yes                     No
 Stornoway           EGPO    2014    Yes                     No                      No
 Blackpool           EGNH    2014    Yes                     Yes                     No
 Lands End           EGHC    2014    Yes                     No                      No
-London Stansted     EGSS    2014    Yes                     Yes                     Yes
\ No newline at end of file
+London Stansted     EGSS    2014    Yes                     Yes                     Yes
+Dublin				EIDW	2014	Yes						No 						NO
```

Comment:
```suggestion
Dublin              EIDW    2014    Yes                     No                      No
```
Use spaces rather than tabs as tabs may display differently between text editors

Reply:
Don't need this line if there is nothing to update - this doc was only to keep track of where track changes would affect other things in the sector file, which this change won't

Reply:
removed entry altogether